### PR TITLE
Get the privacy level of a user.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: mozillians/dino_park/utils.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class UserAccessLevel(object):
+    """Class to handle privacy related scopes in DinoPark."""
+    # Privacy classifications for Dino Park
+    PUBLIC_DINOS = 'public'
```
Comment (johngian) [inline mozillians/dino_park/utils.py]: Please skip ``DINOS`` and ``DINO`` since
* Don't provide any usability
* Ending changes and it is prone to errors

Comment (johngian) [reply mozillians/dino_park/utils.py]: This has other references

File: mozillians/dino_park/utils.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        """Return user privacy clearance for Dino Park."""
+        request_user = request.user
+
+        if request_user.is_authenticated():
+            if request_user.userprofile.is_vouched:
+                return cls.VOUCHED_DINOS
+            if request_user.userprofile.is_nda:
+                return cls.NDA_DINOS
+            # TODO: This needs to change to hris assertion from ES
+            if user.userprofile.groups.filter(name='staff').exists():
```
Comment (johngian) [inline mozillians/dino_park/utils.py]: Why aren't you using ``request_user`` here ?

Comment (johngian) [reply mozillians/dino_park/utils.py]: ``user`` is going to be ``None`` by default

Comment (akatsoulas) [reply mozillians/dino_park/utils.py]: as per todo - this is a placeholder

File: mozillians/dino_park/utils.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        if request_user.is_authenticated():
+            if request_user.userprofile.is_vouched:
+                return cls.VOUCHED
+            if request_user.userprofile.is_nda:
+                return cls.NDA
+            # TODO: This needs to change to hris assertion from ES
+            if request_user.userprofile.groups.filter(name='staff').exists():
+                return cls.STAFF
+            if (request_user.is_superuser or (user and user == request_user)):
+                return cls.PRIVATE
```
Comment (johngian) [inline mozillians/dino_park/utils.py]: Please add a comment to mention that superusers have super powers.

File: mozillians/dino_park/utils.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    STAFF = 'staff'
+    PRIVATE = 'private'
+
+    @classmethod
+    def get_privacy(cls, request, user=None):
+        """Return user privacy clearance for Dino Park."""
+        request_user = request.user
+
+        if request_user.is_authenticated():
+            if request_user.userprofile.is_vouched:
```
Comment (johngian) [inline mozillians/dino_park/utils.py]: Let's figure out the order of the cases to provide the highest level.

**Pull request merged**